config/http.conf

```diff
@@ -39,6 +46,7 @@ http {
   proxy_set_header    X-Forwarded-Proto   $proxy_x_forwarded_proto;
   proxy_set_header    X-Forwarded-Ssl     $proxy_x_forwarded_ssl;
   proxy_set_header    X-Forwarded-Port    $proxy_x_forwarded_port;
+  proxy_set_header    X-Request-ID        $proxy_x_request_id;
```

Comment:
Some apps [like Rails with use this as the `request_id` if the `X-Request-ID` exists](https://api.rubyonrails.org/classes/ActionDispatch/RequestId.html). Else, they generate one. So it's certainly useful to set it here because it will be used upstream. See ...

<img width="1072" alt="Screenshot 2021-02-10 at 12 01 48" src="https://user-images.githubusercontent.com/703786/107487947-c6887f00-6b97-11eb-8539-9034b444e116.png">